Add hardware docs (#68)
Reznic/Inhalator: docs/README.md

# Software Configuration

1. This guide will document the requierments on the software.

> 2. The software accesses the reading of 3 sensors, oxygen-sensor (via a2d) , battery-level(via a2d)  and flow-sensor (directly).

Reviewer: We also reading the pressure sensor....
```suggestion
2. The software accesses the reading of 4 sensors:
    1. Oxygen (via a2d)
    2. Battery Level (via a2d)
    3  Air Flow (I2C)
    4. Pressure (I2C)
```

> 3. The software handles and RTC which will raise a WD alarm which causes a software reset

Reviewer: The WD RTC does not reset the software automatically
```suggestion
3. The software also "kicks" an RTC Watchdog which will sound an alarm if the software hangs
```

> 4. The software controls LEDs and turn them on in case of an alaram

Reviewer: Typo in alarm, better description.
```suggestion
4. The software controls LEDs and turn them on/off reflecting the alerts in the system 
```

The software accesses several data buses as described later.
The software-hardware interfaces is described later. 

## LOGIC 

### SENSOR READ LOGIC 
Read A2D CH0 every 1 seconds
Read A2D CH1 every 15 minutes

> Read Flow-Sensor every 1 seconds

Reviewer: The intervals are not correct...

### WD LOGIC 
Set WD alarm to 30 seconds, reset the alarm before the WD rises. 

> If WD signal is raised the reset the software

Reviewer: ```suggestion
If WD signal is raised - sound an alarm
```
